pennylane/tape/qubit_param_shift.py

```diff
@@ -379,3 +379,22 @@ def processing_fn(results):
             return np.apply_along_axis(dot, 0, results)
 
         return tapes, processing_fn
+
+    @property
+    def specs(self):
+
+        if "grad_method" not in self._par_info[0]:
+            self._update_gradient_info()
+
+        # Initialize with the forward pass execution
+        num_executions = 1
+        # Loop over all variables
+        for _, info in self._par_info.items():
+
+            if info["grad_method"] == "A":

```

Comment:
How about finite diff and CV circuits? Maybe not too important for now, but might be worth considering later.

Reply:
I've thought about it a little and will probably try to add them in later.